docs/README.md

```diff
@@ -1,42 +1,154 @@
-# WatermelonDB Documentation
+<p align="center">
```

Comment:
can we maybe add a `cp` to the travis script, so that we don't have to maintain a copy of README?

Reply:
we could do the same with changelog and contributing maybe? Which I think should stay in the root because some tools auto-recognize such files

Reply:
good idea